seatunnel-connectors-v2/connector-iceberg/src/main/java/org/apache/seatunnel/connectors/seatunnel/iceberg/config/CommonConfig.java

```diff
@@ -20,27 +20,62 @@
 import static com.google.common.base.Preconditions.checkArgument;
 import static com.google.common.base.Preconditions.checkNotNull;
 
+import org.apache.seatunnel.api.configuration.Option;
+import org.apache.seatunnel.api.configuration.Options;
+
 import org.apache.seatunnel.shade.com.typesafe.config.Config;
 
 import lombok.Getter;
 import lombok.ToString;
 
 import java.io.Serializable;
+import java.util.List;
 
 @Getter
 @ToString
 public class CommonConfig implements Serializable {
     private static final long serialVersionUID = 239821141534421580L;
 
-    private static final String KEY_CATALOG_NAME = "catalog_name";
-    private static final String KEY_CATALOG_TYPE = "catalog_type";
-    private static final String KEY_NAMESPACE = "namespace";
-    private static final String KEY_TABLE = "table";
-    private static final String KEY_URI = "uri";
-    private static final String KEY_WAREHOUSE = "warehouse";
-    private static final String KEY_CASE_SENSITIVE = "case_sensitive";
+    public static final Option<String> KEY_CATALOG_NAME = Options.key("catalog_name")
+        .stringType()
+        .noDefaultValue()
+        .withDescription(" the iceberg catalog name");
+
+    public static final Option<String> KEY_CATALOG_TYPE = Options.key("catalog_type")
```

Comment:
I think make `catalog_type` as an enum type is better.

Reply:
@hailin0  It's good to do this. What do you think?

Reply:
> @hailin0 It's good to do this. What do you think?

agree with you